app/models/user.rb

```diff
@@ -9,6 +9,14 @@ class User < ApplicationRecord
   validates :password, presence: true, length: { minimum: Settings.user.validates.password.min }, on: :create
   validates :password_confirmation, presence: true, on: :create
 
+  before_create :create_remember_token
+
   enum role: %i(user admin)
   enum status: %i(enabled disabled)
+
+  private
+
+  def create_remember_token
+    self.remember_token = SecureRandom.urlsafe_base64
```

Comment:
Mỗi lần login thì remember_token  của user có thay đổi ko e?

Reply:
@thanhtt-3322 thế thì có àn toàn ko. user xem remember_token trong cookies rồi tự set lại cái đăng nhập được luôn à ?

Reply:
Dạ vậy e generate lại token mỗi lần đăng nhập